tests/migration/test_aura_contracts_migration.py

```diff
@@ -0,0 +1,137 @@
+import brownie
+import time
+import pytest
+
+from brownie import interface, accounts, StrategyAuraStaking, AdminUpgradeabilityProxy, Contract, chain
+from helpers.constants import AddressZero, MaxUint256
+from helpers.time import days
+from rich.console import Console
+
+C = Console()
+
+CURRENT_BOOSTER = "0x7818A1DA7BD1E64c199029E86Ba244a9798eEE10"
+NEW_BOOSTER = "0xA57b8d98dAE62B26Ec3bcC4a365338157060B234"
+CURRENT_PIDS = {
+    "b80BADGER_20WBTC": 33,
+    "b40WBTC_40DIGG_20graviAURA": 34
+}
+NEW_PIDS = {
+    "b80BADGER_20WBTC": 18,
+    "b40WBTC_40DIGG_20graviAURA": 19
+}
+VAULTS = {
+    "b80BADGER_20WBTC": "0x63ad745506BD6a3E57F764409A47ed004BEc40b1",
+    "b40WBTC_40DIGG_20graviAURA": "0x371B7C451858bd88eAf392B383Df8bd7B8955d5a"
+}
+STRATEGIES = {
+    "b80BADGER_20WBTC": "0xe43857fE16D18b6633A663389934d6c64D5E81FD",
+    "b40WBTC_40DIGG_20graviAURA": "0xD87F2cdE238D0122b3865164359CFF6b2431d927"
+}
+
+LAST_PERIOD_FINISH = 1671696167
+
+@pytest.mark.parametrize('VAULT_ID', ["b80BADGER_20WBTC", "b40WBTC_40DIGG_20graviAURA"])
+def test_aura_contracts_migration(VAULT_ID):
+    C.log(f"Migration test for: {VAULT_ID}")
+    vault = interface.IVault(VAULTS[VAULT_ID])
+    gov = accounts.at(vault.governance(), force=True)
+    strat_current = StrategyAuraStaking.at(STRATEGIES[VAULT_ID], owner=gov)
+    want = interface.ERC20(vault.token())
+    assert vault.strategy() == strat_current.address
+
+    # Aura contracts
+    assert strat_current.BOOSTER() == CURRENT_BOOSTER
+    booster_current = interface.IBooster(CURRENT_BOOSTER)
+
+    # Fast forward to end of rewards period for current contracts
+    C.print(f"Sleeping for {(LAST_PERIOD_FINISH - chain.time())/(days(1))} days")
+    chain.sleep(LAST_PERIOD_FINISH - chain.time())
```

Comment:
Better to mine after sleep to make sure `eth_call` values aren't stale
```suggestion
    chain.sleep(LAST_PERIOD_FINISH - chain.time())
    chain.mine()
```
Explains the discrepancy between the earned and harvested amount
<img width="326" alt="image" src="https://user-images.githubusercontent.com/27727946/208099319-0d5da273-204f-42d2-a1b1-e2345df7e313.png">

Reply:
Very good call, I didn't get a chance yesterday to figure that out and missed it. Thanks for the heads up, pushed.